projects/core/src/routing/facade/routing.service.ts

```diff
@@ -79,6 +81,27 @@ export class RoutingService {
     return this.navigate(path, query, extras);
   }
 
+  /**
+   * Resolves the absolute url for the given `UrlCommands`.
+   *
+   * The absolute URL uses the origin of the current location.
+   */
+  getUrl(commands: UrlCommands): string {
```

Comment:
To be more reusable - I'd call it `getAbsoluteUrl`.
I wouldn't put there the origin (or at least create a separate method for it if needed), i.e. `getFullUrl`

Reply:
I'm putting this on hold to first see how we conclude the usage of the SSR provided values (href/origin). 
`getUrl` worked for me, and I thought we'd implement `getRelativeUrl` or `getRoutePath` over time when needed. At the same time i'm not against `getFullUrl`as an alternative.

Reply:
Good feedback. I've followed your approach, but combined it with the earlier suggestion of having `getFullUrl`, just to simplify the arguments.